test/metabase/api/search_test.clj

```diff
@@ -96,7 +103,7 @@
     test-collection
     (make-result "card test card", :model "card", :bookmark false, :dataset_query nil, :dashboardcard_count 0)
     (make-result "dataset test dataset", :model "dataset", :bookmark false, :dataset_query nil, :dashboardcard_count 0)
-    (make-result "action test action", :model "action", :model_name (:name action-model-params), :model_id true)
+    (make-result "action test action", :model "action", :model_name (:name action-model-params), :model_id true, :dataset_query (update (mt/query venues) :type name))
```

Comment:
The search-result-making code is getting increasingly crufty. "rewrite the search tests" is ratcheting up my priority list, but…not today.